main.py

```diff
@@ -48,8 +48,9 @@ def load_stache(stache_name, possible_dirs):
         path = Path(dir_).resolve()
         python_path = path.parent
         with PythonPathContext(str(python_path)):
-            module_spec = importlib.util.find_spec(
-                "{}.{}".format(str(find_last_child(path)), stache_name))
+            module_spec = importlib.util.spec_from_file_location(
+                    "{}.{}".format(str(find_last_child(path)), stache_name),
+                    "{}\{}.py".format(str(find_last_child(path)),stache_name))
```

Comment:
This should be a forward slash not a backslash.

You can fix this with `git commit --amend`.